docs/application/native/guides/app-management/app-controls.md

```diff
@@ -63,18 +69,18 @@ You can launch an application with an application control using 2 different laun
 <a name="request"></a>
```

Comment:
Line 67:
Kindly rephrase the line as "...launch service applications outside the current package, are not supported anymore. Because of this..."

Reply:
@hjhun Kindly incorporate this change too.

Reply:
@mijin85cho Yes, because while reading, it is confusing whether the services are allowed or not since Tizen 2.4.